util/mkerr.pl

```diff
@@ -6,325 +6,314 @@
-                the quotes, naturally)
-                NOTE: not used any more because our include directory
-                structure has changed.
-
-  -debug        Turn on debugging verbose output on stderr.
-
-  -rebuild      Rebuild all header and C source files, irrespective of the
-                fact if any error or function codes have been added/removed.
-                Default: only update files for libraries which saw change
-                         (of course, this requires '-write' as well, or no
-                          files will be touched!)
-
-  -recurse      scan a preconfigured set of directories / files for error and
-                function codes:
-                  (<crypto/*.c>, <crypto/*/*.c>, <ssl/*.c>, <apps/*.c>)
-                When this option is NOT specified, the filelist is taken from
-                the commandline instead. Here, wildcards may be embedded. (Be
-                sure to escape those to prevent the shell from expanding them
-                for you when you wish mkerr.pl to do so instead.)
-                Default: take file list to scan from the command line.
-
-  -reindex      Discard the numeric values previously assigned to the error
-                and function codes as extracted from the scanned header files;
-                instead renumber all of them starting from 100. (Note that
-                the numbers assigned through 'R' records in the config file
-                remain intact.)
-                Default: keep previously assigned numbers. (You are warned
-                         when collisions are detected.)
-
-  -nostatic     Generates a different source code, where these additional
-                functions are generated for each library specified in the
-                config file:
-                  void ERR_load_<LIB>_strings(void);
-                  void ERR_unload_<LIB>_strings(void);
-                  void ERR_<LIB>_error(int f, int r, char *fn, int ln);
-                  #define <LIB>err(f,r) ERR_<LIB>_error(f,r,OPENSSL_FILE,OPENSSL_LINE)
-                while the code facilitates the use of these in an environment
-                where the error support routines are dynamically loaded at
-                runtime.
-                Default: 'static' code generation.
+    -conf file  Use the named config file F instead of the default.
+
+    -debug      Verbose output debugging on stderr.
+
+    -rebuild    Force a rebuild all header and C source files, even if there
+                were no changes.
+
+    -recurse    scan a preconfigured set of files for error and function codes.
+                If not specified, files on commandline are used.
+
+    -reindex    Ignore previously assigned values (except for R records in
+                the config file and renumber everything starting at 100.
+
+    -nowrite    Do not rewrite the header/source files.
+
+    -internal   Generate code that is to be built as part of OpenSSL itself.
```

Comment:
This set of options has added indentation.  Dunno if that was intentional, but try to have them all be consistent.